pkg/liquidity-mining/test/AuthorizerAdaptorEntrypoint.test.ts

```diff
@@ -151,6 +152,10 @@ describe('AuthorizerAdaptorEntrypoint', () => {
       });
     });
 
+    it('rejects direct calls from the adaptor', async () => {
```

Comment:
No - but it is still wrong, because the test above it changed. It's trying to test that an otherwise correct call - that succeeds when called on the entrypoint - fails when called on the adaptor.